src/uct/sm/mm/base/mm_iface.c

```diff
@@ -64,6 +64,17 @@ ucs_config_field_t uct_mm_iface_config_table[] = {
     {NULL}
 };
 
+ucs_config_field_t uct_mm_ext_iface_config_table[] = {
```

Comment:
can't we add this to uct_mm_iface_config_table? why need a separate structure?

Reply:
for XPMEM not to show error handling config

Reply:
IMO we can expose in XPMEM as well

Reply:
fixed